app/ui-react/packages/ui/src/Layout/AppLayout.tsx

```diff
@@ -77,9 +77,15 @@ export const AppLayout: React.FunctionComponent<ILayoutBase> = ({
   const onNavToggleMobile = () => {
     setIsNavOpenMobile(!isNavOpenMobile);
   };
+  const [curViewportWidth, setCurViewportWidth] = React.useState(1024);
   const onPageResize = (props: { mobileView: boolean; windowSize: number }) => {
     setIsMobileView(props.mobileView);
+    setCurViewportWidth(props.windowSize);
   };
+  const [isTabletView, setIsTabletView] = React.useState(false);
+  React.useEffect(() => {
+    setIsTabletView(curViewportWidth <= 992);
```

Comment:
What's this number?  Seems kinda arbitrary.

Reply:
Ah yea so it's actually a common breakpoint, we use it in core for denoting a "large" viewport - https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/sass-utilities/scss-variables.scss#L154

I'm looking now to see if we can pull this value dynamically from one of the styling packages.

Reply:
TBH even just naming it with a const would be good like `const SUPER_IMPORTANT_BREAKPOINT=992` and then use that, then it's kinda self documenting :-)

Reply:
Can do!